app/src/main/java/com/example/mycalculator/utils/StringUtils.java

```diff
@@ -2,5 +2,6 @@
 
 public class StringUtils {
```

Comment:
you can unify StringUtils and ConstantsUtils class

Reply:
Done